### PR TITLE
Added Retendo Network Service

### PR DESCRIPTION
**Pull request by 00cedke**
I added Retendo Network Service, for play online with Retendo Network, [The API will soon work].

### CODE REVIEW

**Comment by qurious-pixel** [comment]
@00cedke , why would you change the official repo to a fork in the build instructions?

File: BUILD.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Prerequisites:
 
 Instructions for Visual Studio 2022:
 
-1. Run `git clone --recursive https://github.com/cemu-project/Cemu`
+1. Run `git clone --recursive https://github.com/RetendoNetwork/Cemu`
```
Comment (qurious-pixel) [inline BUILD.md:35]: revert back to `git clone --recursive https://github.com/cemu-project/Cemu`

File: BUILD.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ At Step 3 in [Build Cemu using cmake and clang](#build-cemu-using-cmake-and-clan
 #### CMake and Clang
 
 ```
-git clone --recursive https://github.com/cemu-project/Cemu
+git clone --recursive https://github.com/RetendoNetwork/Cemu
```
Comment (qurious-pixel) [inline BUILD.md:67]: revert back to `git clone --recursive https://github.com/cemu-project/Cemu`

File: BUILD.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ If you are building using GCC, make sure you have g++ installed:
 - Installation for Fedora and derivatives: `sudo dnf install gcc-c++`
 
 ```
-git clone --recursive https://github.com/cemu-project/Cemu
+git clone --recursive https://github.com/RetendoNetwork/Cemu
```
Comment (qurious-pixel) [inline BUILD.md:81]: revert back to `git clone --recursive https://github.com/cemu-project/Cemu`

File: BUILD.md [hunk 4]
```diff
@@ -87,7 +87,7 @@ cmake --build build
 #### Debug Build
 
 ```
-git clone --recursive https://github.com/cemu-project/Cemu
+git clone --recursive https://github.com/RetendoNetwork/Cemu
```
Comment (qurious-pixel) [inline BUILD.md:90]: revert back to `git clone --recursive https://github.com/cemu-project/Cemu`

File: BUILD.md [hunk 5]
```diff
@@ -173,7 +173,7 @@ Then install the dependencies:
 
 ### Build Cemu using CMake
 
-1. `git clone --recursive https://github.com/cemu-project/Cemu`
+1. `git clone --recursive https://github.com/RetendoNetwork/Cemu`
```
Comment (qurious-pixel) [inline BUILD.md:176]: revert back to `git clone --recursive https://github.com/cemu-project/Cemu`

**Comment by 00cedke** [comment]
So that people compile the Cemu with the changes I made in case if they don't put Retendo on Cemu, but if they do, I will put the normal repository, so if you want me to put it back now

**Comment by 00cedke** [comment]
I changed, sorry.

**Comment by Exzap** [comment]
Since you don't have a website yet and the service isn't actually running from what I can tell, it's a bit premature to add this option to Cemu already

**Comment by 00cedke** [comment]
No problem, sorry for the inconvenience.

**Pull request closed without merging**